exonum-java-binding/core/rust/src/utils/errors.rs

```diff
@@ -176,6 +177,20 @@ pub fn unwrap_exc_or_default<T: Default>(env: &JNIEnv, res: ExceptionResult<T>)
     unwrap_exc_or(env, res, T::default())
 }
 
+/// Returns a cause of the passed exception by using `Throwable#getCause` method.
+///
+/// Panics if `exception` is null.
```

Comment:
It is perfectly acceptable for an exception to have `null` cause; only UTEE cannot
have `null` cause, therefore I'd reconsider such behaviour in a seemingly general purpose method.

Reply:
Good catch, but it panics when passed exception is null, not its cause is null. So I'd add a check for null cause in the appropriate places, but this particular method seems okay.
Another option might be returning `Option` to represent empty cause, I'd consider that.

Reply:
I missed that, sorry. 

> for null cause 

I don't think it is needed much because UTEE _enforces_ that in its ctor.